src/lib/containers/FullWidthAlert.tsx

```diff
@@ -1,10 +1,9 @@
 import React, { useEffect } from 'react'
 import { Button, Alert, AlertProps } from 'react-bootstrap'
 import {
-  InfoOutlined,
-  ErrorOutlined,
-  WarningOutlined,
-  CheckCircleOutlined,
+  InfoTwoTone,
```

Comment:
Fix icons (use two tone versions, and use ErrorTwoTone for both warning and error)